django/core/handlers/exception.py

```diff
@@ -134,6 +124,17 @@ def response_for_exception(request, exc):
             response = get_exception_response(
                 request, get_resolver(get_urlconf()), 400, exc
             )
+        # The logger is set to django.security, which specifically captures
+        # SuspiciousOperation events, unlike the default django.request logger.
+        security_logger = logging.getLogger(f"django.security.{exc.__class__.__name__}")
+        log_response(
+            str(exc),
+            exception=exc,
+            request=request,
+            response=response,
+            level="error",
+            logger=security_logger,
+        )
```

Comment:
I guess, is this change needed or perhaps should this be a separate commit?
This doesn't log the `request.path` and so I don't think it's vulnerable (the added test is also passing on main).
Perhaps this should be a separate cleanup commit if we want to keep it?